hamilton/driver.py

```diff
@@ -726,8 +789,12 @@ def display_upstream_of(
                 output_file_path,
                 render_kwargs=render_kwargs,
                 graphviz_kwargs=graphviz_kwargs,
-                strictly_display_only_passed_in_nodes=False,
+                strictly_display_only_passed_in_nodes=True,
```

Comment:
why this change?

Reply:
@zilto we should add a comment. I think this is a no-op -- anything upstream will be covered by that. I think we'll want to remove that paramter entirely -- we can calculate it better outside of the function.